src/trusted/service_runtime/sel_ldr.c

```diff
@@ -670,6 +704,10 @@ int32_t NaClSetAvail(struct NaClApp  *nap,
   return pos;
 }
 
+int NaClGetNumThreads(struct NaClApp        *nap) {
+  return (&nap->threads)->num_entries;
+}
```

Comment:
I think I'd rather just access the num_entries field directly than have this function, or at least have it inlined in a header or something.